proto/quicksilver/interchainstaking/v1/query.proto

```diff
@@ -143,6 +151,14 @@ message QueryDelegatorIntentResponse {
   DelegatorIntent intent = 1;
 }
 
+message QueryDelegatorIntentsRequest {
+  string delegator_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+message QueryDelegatorIntentsResponse {
+  repeated DelegatorIntent intents = 1;
```

Comment:
Does the DelegatorIntent contain the chain Id?

Reply:
It does not.

We should use something like: 

```proto
message DelegatorIntentsResponse {
     string chainID = 1;
     DelegatorIntent intent = 2;
}

message QueryDelegatorIntentsResponse {
  repeated DelegatorIntentsResponse intents = 1;
```

Reply:
added a chain id to the response as well with each intent